components/dueling-picklist/__examples__/automatic-height-minimization.jsx

```diff
@@ -0,0 +1,56 @@
+/* eslint-disable no-console, react/prop-types */
+import React from 'react';
+import IconSettings from '~/components/icon-settings';
+import DuelingPicklist from '~/components/dueling-picklist';
+import duelingPicklistFilter from '~/components/dueling-picklist/filter';
```

Comment:
This needs to be exported in the `components/index.js`. Examples only have access to exports in that file.